entityDumper/dumper.go

```diff
@@ -0,0 +1,37 @@
+package entityDumper
+
+import (
+	"bufio"
+	"github.com/rs/zerolog/log"
+	"github.com/uyuni-project/inter-server-sync/schemareader"
+	"os"
+)
+
+func DumpAllEntities(options DumperOptions) {
+	var outputFolderAbs = options.GetOutputFolderAbsPath()
+	validateExportFolder(outputFolderAbs)
+	db := schemareader.GetDBconnection(options.ServerConfig)
+	defer db.Close()
+
+	channelsExport := loadChannelsToProcess(db, options)
```

Comment:
Yeah, it fits well under the if statement, so that we have different sessions for each resource being exported. However I don't see any practical benefit of getting rid of the dependency injection and doing the initialization inside the function itself. But of course I don't have the overview you got, so please tell if if I'm wrong :smiley: